src/CssInliner.php

```diff
@@ -61,6 +61,11 @@ class CssInliner extends AbstractHtmlProcessor
      */
     private $excludedSelectors = [];
 
+    /**
+     * @var array<string, bool>
```

Comment:
AFAICT, we can even use `non-empty-string` instead of `string` here.

Reply:
Actually, no. Because `array<non-empty-string, bool>` is a child of `non-empty-array` and will raise analysis errors.

Reply:
I'm confused - are you by change thinking of `non-empty-array<string, bool>` instead of `array<non-empty-string, bool>`? (As far as I understand it, the latter would also allow an empty array.)

Reply:
> As far as I understand it, the latter would also allow an empty array

Seems like it doesn't, here's the output of Psalm:
```
ERROR: PropertyTypeCoercion - src/CssInliner.php:318:9 - $this->excludedCssSelectors expects 'array<non-empty-string, bool>',  parent type 'non-empty-array<string, bool>' provided (see https://psalm.dev/198)
        $this->excludedCssSelectors[$selector] = true;
```

Reply:
Error thrown using:
```php
    /**
     * @var array<non-empty-string, bool>
     */
    private $excludedCssSelectors = [];
```

Reply:
Ok found it, `add/removeExcludedCssSelector` also has to have `non-empty-string`, not just string (which can be empty).